smart-contracts/contracts/mixins/MixinLockCore.sol

```diff
@@ -139,8 +141,8 @@ contract MixinLockCore is
     uint _keyPrice,
     address _tokenAddress
```

Comment:
Hmm we should probably add a comment that address(0) is used for ETH.

Note that there are other standards out there - such as 0xeeeeeee... for eth.

Reply:
Yup.done.